minibot.py

```diff
@@ -0,0 +1,272 @@
+"""Contains modifications and extensions to RobotPy
+ needed for the minibot project"""
+
+import wpilib
+import hal
+import networktables
+
+import pwmoutput
+import micro_maestro_interface
+
+
+class Minibot(wpilib.SampleRobot):
+    """Subclass of wpilib.SampleRobot, handles Pololu board, etc."""
+
+    def __init__(self, *args, **kwargs):
+        super(Minibot, self).__init__(*args, **kwargs)
+
+        self.ds = DriverStation()
+
+        try:
+            micro_maestro = micro_maestro_interface.MicroMaestro("test", 0)
+        except RuntimeError:
+            # No micro maestro on correct port
+            micro_maestro = None
+
+        pwmoutput.PwmWatch(micro_maestro)
+
+    def startCompetition(self):
```

Comment:
Can you add a comment to this function describing where it came from? How is it different from the wpilib.SampleRobot impl or where did you copy it from? There's nothing obvious that makes it clear why its in this class.

Reply:
Will do, that's a good point.